aurora-rust-sdk/aurora-sdk-integration-tests/src/lib.rs

```diff
@@ -0,0 +1,13 @@
+pub use aurora_engine_sdk;
+pub use aurora_engine_types;
+pub use ethabi;
+pub use tokio;
+pub use workspaces;
+
+pub mod aurora_engine;
+pub mod nep141;
+pub mod utils;
+pub mod wnear;
+
+#[cfg(test)]
+mod tests;
```

Comment:
The motivation for this is probably the same as for having `integration-tests` in the example. Maybe it's just me, but I'm thinking of `tests` as kind of a reserved name for directories in Rust projects, so I think a short comment could be helpful. Though feel free to ignore it if this usage of `tests` is a common pattern.

Reply:
Added a link to a blog post which discusses this pattern in https://github.com/aurora-is-near/aurora-contracts-sdk/pull/2/commits/55bd2839ee8a8d626362193194e78c5bcd74d040